### PR TITLE
update installation instruction

### LINKED ISSUE #3756: pynio package not found
Copied the install instructions from the docs http://xarray.pydata.org/en/stable/installing.html#instructions on my windows machine.
`conda install -c conda-forge xarray cartopy pynio pseudonetcdf`

Got a `PackagesNotFoundError: pynio`

image

### PR DESCRIPTION
**Pull request by mathause**
- [x] Closes #3756
 - [x] Fully documented, including `whats-new.rst` for all changes and `api.rst` for new API

### CODE REVIEW

**Comment by mathause** [comment]
I thought it best to remove the instructions
``` bash
$ conda install -c conda-forge xarray cartopy pynio pseudonetcdf
```
completely - else people are going to copy-paste the line regardless of what's written in the text. Most people won't need *these* specific packages.

@raybellwaves, @keewis

File: doc/installing.rst [hunk 1]
```diff
@@ -121,16 +121,15 @@ xarray itself is a pure Python package, but its dependencies are not. The
 easiest way to get everything installed is to use conda_. To install xarray
 with its recommended dependencies using the conda command line tool::
 
-    $ conda install xarray dask netCDF4 bottleneck
+    $ conda install -c conda-forge xarray dask netCDF4 bottleneck
 
 .. _conda: http://conda.io/
 
-We recommend using the community maintained `conda-forge <https://conda-forge.github.io/>`__ channel if you need difficult\-to\-build dependencies such as cartopy, pynio or PseudoNetCDF::
+If you require other `optional dependencies <#Optional dependencies>`_ add them to the line above.
```
Comment (keewis) [inline doc/installing.rst]: this link looks strange to me and I think it does not point anywhere. To fix it, I'd add a label in front of the `Optional dependencies` heading and then reference that:
```rst
`optional dependencies <optional-dependencies>`_
```
or
```rst
:doc:`optional-dependencies`
```

**Comment by mathause** [comment]
Thanks for the quick feedback. I found somewhere that you can directly link to titles, but this is more explicit.

**Comment by raybellwaves** [comment]
LGTM

**Comment by max-sixty** [comment]
:+1:

**Comment by keewis** [comment]
I didn't know that, I thought that was limited to `nbsphinx`. It did work, though, the link pointed to some anchor; you just specified the wrong name.

I think you can also use a mixture of both:
```rst
:ref:`optional dependencies <optional-dependencies>`
```

Also, using `:ref:` should fix the build error.

**Comment by keewis** [comment]
the docs build passed. Thanks, @mathause

**Pull request merged**